### PR TITLE
Feature/dfget param

### LINKED ISSUE #372: adjust dfget param
### Feature request:

adjust dfget param

### PR DESCRIPTION
**Pull request by garfield009**
## Description
adjust dfget grpc param

## Related Issue
fix #372 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by 244372610 — APPROVED** [review]

File: internal/rpc/dfdaemon/dfdaemon.proto [hunk 1]
```diff
@@ -24,21 +24,26 @@ import "google/protobuf/empty.proto";
...
-  // caller business id
-  string biz_id = 4;
-  // regex format
-  string filter = 5;
-  // identify one downloading
-  // framework will fill it automatically
-  string uuid = 6;
-  int64 uid = 7;
-  int64 gid = 8;
+  string output = 3;
```
Comment (244372610) [inline internal/rpc/dfdaemon/dfdaemon.proto:33]: add ratelimit param ?

File: internal/rpc/dfdaemon/dfdaemon.proto [hunk 2]
```diff
@@ -24,21 +24,26 @@ import "google/protobuf/empty.proto";
...
+  int64 timeout = 4;
+  // rate limit in bytes per second
+  int64 limit = 5;
+  base.UrlMeta url_meta = 6;
+  // p2p/cdn/source
+  string pattern = 7;
+  // call system
+  string callsystem = 8;
+  // user id
+  int64 uid = 9;
```
Comment (gaius-qi) [inline internal/rpc/dfdaemon/dfdaemon.proto:44]: Use `uint64`.

File: internal/rpc/dfdaemon/dfdaemon.proto [hunk 3]
```diff
@@ -24,21 +24,26 @@ import "google/protobuf/empty.proto";
...
+  // group id
+  int64 gid = 10;
```
Comment (gaius-qi) [inline internal/rpc/dfdaemon/dfdaemon.proto:46]: Use `uint64`.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: internal/rpc/dfdaemon/dfdaemon.proto (hunk 2 above)
Comment (garfield009) [reply internal/rpc/dfdaemon/dfdaemon.proto:44]: no need change it, interface need int64

File: internal/rpc/dfdaemon/dfdaemon.proto (hunk 3 above)
Comment (garfield009) [reply internal/rpc/dfdaemon/dfdaemon.proto:46]: no need change it, interface need int64

**Review by gaius-qi — APPROVED** [review]

**Pull request merged**